pkgs/development/python-modules/commitizen/default.nix

```diff
@@ -80,16 +84,6 @@ buildPythonPackage rec {
 
   pythonImportsCheck = [ "commitizen" ];
 
-  # The tests require a functional git installation
-  # which requires a valid HOME directory.
-  preCheck = ''
-    export HOME="$(mktemp -d)"
-
-    git config --global user.name "Nix Builder"
-    git config --global user.email "nix-builder@nixos.org"
-    git init .
```

Comment:
This part is not done by the hook. Not sure if it was actually load bearing?

Reply:
Nice catch, fixed.